spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/mongo/ReactiveMongoClientFactoryTests.java

```diff
@@ -117,13 +118,13 @@ void retryWritesIsPropagatedFromUri() {
 	}
 
 	@Test
-	void uriCannotBeSetWithCredentials() {
+	void uriCanBeSetWithCredentials() {
```

Comment:
I don't think we should be making that change as part of the upgrade to the new driver unless this use case is no longer supported.